[2024-04-25T07:54:33.381+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-25T07:54:33.402+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v01.insert_into_table scheduled__2022-02-09T00:00:00+00:00 [queued]>
[2024-04-25T07:54:33.410+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v01.insert_into_table scheduled__2022-02-09T00:00:00+00:00 [queued]>
[2024-04-25T07:54:33.414+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2024-04-25T07:54:33.440+0000] {taskinstance.py:2327} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2022-02-09 00:00:00+00:00
[2024-04-25T07:54:33.458+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1003) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-25T07:54:33.460+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v01', 'insert_into_table', 'scheduled__2022-02-09T00:00:00+00:00', '--job-id', '2867', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmp7h3oab2c']
[2024-04-25T07:54:33.465+0000] {standard_task_runner.py:91} INFO - Job 2867: Subtask insert_into_table
[2024-04-25T07:54:33.463+0000] {standard_task_runner.py:63} INFO - Started process 1012 to run task
[2024-04-25T07:54:33.534+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_with_postgres_operator_v01.insert_into_table scheduled__2022-02-09T00:00:00+00:00 [running]> on host 4d994982806b
[2024-04-25T07:54:33.605+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v01' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2022-02-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-02-09T00:00:00+00:00'
[2024-04-25T07:54:33.606+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-25T07:54:33.615+0000] {sql.py:276} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2022-02-09', 'dag_with_postgres_operator_v01')
        
[2024-04-25T07:54:33.624+0000] {base.py:84} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-04-25T07:54:33.632+0000] {base.py:84} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-04-25T07:54:33.642+0000] {sql.py:457} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2022-02-09', 'dag_with_postgres_operator_v01')
        , parameters: None
[2024-04-25T07:54:33.644+0000] {sql.py:466} INFO - Rows affected: 1
[2024-04-25T07:54:33.648+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/providers/common/sql/hooks/sql.py:407 AirflowProviderDeprecationWarning: Call to deprecated method _make_common_data_structure. (The `_make_serializable` method is deprecated and support will be removed in a future version of the common.sql provider. Please update the DbApiHook's provider to a version based on common.sql >= 1.9.1.)
[2024-04-25T07:54:33.651+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-25T07:54:33.662+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v01, task_id=insert_into_table, execution_date=20220209T000000, start_date=20240425T075433, end_date=20240425T075433
[2024-04-25T07:54:33.697+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-25T07:54:33.711+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-25T07:54:33.713+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
